packages/terra-tabs/src/clinical-lowlight-theme/Tab.module.scss

```diff
@@ -1,6 +1,13 @@
 @import '~terra-mixins/lib/Mixins';
 
 :local {
+  .button:focus {
+    &[data-terra-tabs-show-focus-styles='true'] {
+      outline: var(--terra-tabs-add-button-focus-outline, 2px dashed #000);
+      outline-offset: var(--terra-tabs-add-button-focus-outline-offset, -2px);
```

Comment:
we can add `box-shadow: none` here to override focus styles of terra-button. which would prevent the blue highlight around dotted outline

Reply:
Done https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41